nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/pom.xml

```diff
@@ -219,6 +219,11 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.codelibs.elasticsearch.module</groupId>
+      <artifactId>analysis-common</artifactId>
+      <version>6.3.0</version>
```

Comment:
Can we use the `elasticsearch.version` property from the root pom.xml?

Reply:
yes I will move that the main pom thanks

Reply:
:+1: